includes/admin/upgrades/downgrades.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
```

Comment:
Wondering if this needs to be merged into 3.0, or do we need to make a 3.0 issue to remove it (either the entire file, or the pieces that call on it)?